io.kinvolk.Headlamp.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.kinvolk.Headlamp</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Headlamp</name>
+  <summary>An easy-to-use and extensible web UI for Kubernetes</summary>
+  <description>
+    <p>
+      Headlamp is an easy-to-use and extensible Kubernetes web UI.
+      It was created to be a Kubernetes web UI that has the traditional
+      functionality of other web UIs/dashboards available (i.e. to list and view
+      resources) and extra functionality provided by plugins.
+    </p>
+  </description>
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apas02.html -->
+  <categories>
+    <category>Productivity</category>
+    <category>Building</category>
+    <category>Viewer</category>
+    <category>IDE</category>
+    <category>DataVisualization</category>
+  </categories>
+  <url type="homepage">https://github.com/kinvolk/headlamp/</url>
+  <url type="bugtracker">https://github.com/kinvolk/headlamp/issues</url>
+  <url type="help">https://github.com/kinvolk/headlamp#get-involved</url>
+  <launchable type="desktop-id">io.kinvolk.Headlamp.desktop</launchable>
+
+  <releases>
+    <release version="0.3.0" date="2021-04-17" />
+    <release version="0.2.1" date="2021-02-03" />
+    <release version="0.2.0" date="2021-01-14" />
+    <release version="0.1.3" date="2021-11-27" />
```

Comment:
This date is in the future, hence the build failure.

Reply:
Woops! Definitely we didn't do it in 7 months.